README.md

```diff
@@ -17,6 +17,10 @@ Logs in the local Docker client to one or more Amazon ECR registries.
       run: |
         docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
         docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+        
+      - name: Logout of Amazon ECR
+        if: always()
+        run: docker logout ${{ steps.login-ecr.outputs.registry }}
```

Comment:
The indentation looks off to me, it should be aligned with `- name: Build...` above

Reply:
thanks, fixed